app/bundles/DynamicContentBundle/Config/config.php

```diff
@@ -27,7 +27,7 @@
         'public' => [
             'mautic_api_dynamicContent_index' => [
                 'path'       => '/dwc',
-                'controller' => 'Mautic\DynamicContentBundle\Controller\DynamicContentApiController::getEntitiesAction',
+                'controller' => 'Mautic\DynamicContentBundle\Controller\DynamicContentApiController::getAction',
```

Comment:
I'm not sure if this endpoint is used, because `getEntitiesAction` method does not exist in the controller. `getAction` method does exist. So it's a random fix here but I wrote tests for this and it showed this up.